docs: bluetooth: mesh: Add documentation for models shell commands (#7111)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/mesh/gen_battery_cli.rst

The Generic Battery Client model can query the state of a :ref:`bt_mesh_battery_srv_readme` model remotely, but does not have any ability to control the Battery state.

Extended models
****************

None.

Persistent storage
*******************

None.

Shell commands
**************

The Bluetooth mesh shell subsystem provides set of commands to interract with the Generic Battery Client model instantiated on a device.

To make these commands available, enable :kconfig:`CONFIG_BT_MESH_SHELL` and :kconfig:`CONFIG_BT_MESH_SHELL_BATTERY_CLI`.

``mdl_battery instance get-all``
--------------------------------

	Print all instances of the Generic Battery Client model on the device.

``mdl_battery instance set <elem_idx>``
---------------------------------------

> 	Select the Generic Battery Client model instantiated on the specified element id. This instance will be used in message sending. If no model instance was selected, the first model instance found on the device will be used by default.
> 
> 	* ``elem_idx``: Element index where to find the model instance.

Reviewer: ```suggestion
	Select the Generic Battery Client model instantiated on the specified element ID.
	This instance will be used in message sending.
	If no model instance is selected, the first model instance found on the device will be used by default.

	* ``elem_idx``: Element index where the model instance is found.
```
